pandas/tests/window/test_base_indexer.py

```diff
@@ -97,13 +97,57 @@ def get_window_bounds(self, num_values, min_periods, center, closed):
 
 @pytest.mark.parametrize("constructor", [Series, DataFrame])
 @pytest.mark.parametrize(
-    "func,alt_func,expected",
+    "func,np_func,expected,pd_kwargs,np_kwargs",
```

Comment:
We can remove `pd_kwargs` for now since it empty in all these cases?

Reply:
Done.